.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,12 +1,7 @@
 <!--
 * New contributors are highly encouraged to read our
   [CONTRIBUTING](/CONTRIBUTING.md) documentation.
-* The pull request:
-  * Should only fix one issue or add one feature at a time.
-  * Must update the test suite for the relevant functionality.
-  * Should pass all status checks before being reviewed or merged.
-* Commit titles should be prefixed with general area of pull request's change.
```

Comment:
nit but not a blocker: I'd probably leave this part - it's not visible and gives the general idea of what a good PR is like

Reply:
Good point ! Fixed in [657cdec](https://github.com/DataDog/datadog-agent/pull/22487/commits/657cdec87aa10736f940927cf7fb5c13089826b8).